### ISSUE TITLE
[pug] block comment is not working properly

### ISSUE THREAD

**Issue by iagowp** [Microsoft/vscode#55588]
- VSCode Version:
- OS Version:

Steps to Reproduce:

I have this code:
![screen shot 2018-08-01 at 1 18 02 pm](https://user-images.githubusercontent.com/1161036/43534674-5b6ad47a-958e-11e8-9203-94e8b6fa77bd.png)
I want to comment this section:
![screen shot 2018-08-01 at 1 19 03 pm](https://user-images.githubusercontent.com/1161036/43534725-75bd41a0-958e-11e8-9989-2ef6121d3f13.png)
What I get:
![screen shot 2018-08-01 at 1 19 11 pm](https://user-images.githubusercontent.com/1161036/43534783-965f1794-958e-11e8-84c5-151d56597b2d.png)
What I expected:
![screen shot 2018-08-01 at 1 19 34 pm](https://user-images.githubusercontent.com/1161036/43534807-a330c79c-958e-11e8-83a2-ab7ecc6c87e6.png)

Basically, when commenting a block of code, its adding a //- 1 tab before the code, and then adding 1 tab after the //-. I expected it to add a //- at the same tab width (same place?) as the code was

Does this issue occur when all extensions are disabled?: No

Update: I thought I had disable extensions, but I had to reload. Without extensions it worked normally, and after i re-enabled them too. I suspect the problem could have been the Dracula extension, or sublime, but I'm not sure. If I could delete this issue I would =/
